content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -34,21 +34,35 @@ further_reading:
     tag: "GitHub"
     text: "Source code"
 ---
+## Capabilities
+
+- Monitor [runtime metrics][1] to help troubleshoot bugs and detect resource inefficiencies in .NET applications & services.
+
+- Ability to correlate [traces & application logs][2].
+
+- End-to-end visibility with [Real User Monitoring][3] by linking requests from the web and mobile applications to their corresponding backend traces.
 
 ## Compatibility requirements
 
-The .NET Tracer supports instrumentation on:
-  - .NET 5
-  - .NET Core 3.1
-  - .NET Core 2.1
+### Supported .NET Core runtimes
 
-For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
+The .NET Tracer supports instrumentation on .NET Core 2.1 and above.
+
+| Version             
+| -------------------- |
+| .NET 5               |
+| .NET Core 3.1 (LTS)  |
+| .NET Core 3.0        |
+| .NET Core 2.2        |
```

Comment:
We don't support 2.2 and 3.0, why are we adding them to the list?

Reply:
Updated list

Reply:
Updated and reflected in https://github.com/DataDog/documentation/pull/11200/commits/77793e5a94b644a2679fb90ab651acd9a154876b